contracts/GenArt721CoreV3.sol

```diff
@@ -711,23 +713,31 @@ contract GenArt721CoreV3 is
             _additionalPayeeSecondarySales,
             _additionalPayeeSecondarySalesPercentage
         );
-        // automatically accept if no proposed addresses modifications
+        // automatically accept if no proposed addresses modifications,
+        // or if the proposal only removes payee addresses.
         // store proposal hash on-chain, only if not automatic accept
         bool automaticAccept = ((_artistAddress ==
             projectFinance.artistAddress) &&
             (_additionalPayeePrimarySales ==
-                projectFinance.additionalPayeePrimarySales) &&
+                projectFinance.additionalPayeePrimarySales ||
+                _additionalPayeePrimarySales == address(0)) &&
             (_additionalPayeeSecondarySales ==
-                projectFinance.additionalPayeeSecondarySales));
+                projectFinance.additionalPayeeSecondarySales ||
+                _additionalPayeeSecondarySales == address(0)));
```

Comment:
Nit: At this point, I'm tempted to break each of these out into their own bool, for readability purposes, as _I would think_ that this can be easily optimized away to be the same thing by the compiler anyway.

E.g.:

```
bool artistUnchanged = _artistAddress == projectFinance.artistAddress;
bool additionalPrimaryPayeeUnchanged = 
    (_additionalPayeePrimarySales == projectFinance.additionalPayeePrimarySales) ||
        (_additionalPayeePrimarySales == address(0));
bool additionalSecondaryPayeeUnchanged = 
     (_additionalPayeeSecondarySales == projectFinance.additionalPayeeSecondarySales) ||
        (_additionalPayeeSecondarySales == address(0));
bool automaticAccept = 
    artistUnchanged && additionalPrimaryPayeeUnchanged && additionalSecondaryPayeeUnchanged;
```

Reply:
Agree on this, much clearer!
Did cause a compiler stack-to-deep error when implemented as written above, so added block scoping to avoid. Also means optimizer is probably not doing a great job here, but I still think well worth it!